overlay/etc/chromium/master_preferences

```diff
@@ -1,26 +0,0 @@
-{
```

Comment:
Is this chromium prefs file no longer working? Or is it redundant for some reason?